core/domain/rights_manager_test.py

```diff
@@ -629,6 +630,111 @@ def test_get_multiple_exploration_rights(self):
             self.assertIsNotNone(rights_object)
         self.assertIsNone(exp_rights[3])
 
+    def test_owner_cannot_be_reassigned_as_owner(self):
+        exp = exp_domain.Exploration.create_default_exploration(self.EXP_ID)
+        exp_services.save_new_exploration(self.user_id_a, exp)
+
+        with self.assertRaisesRegexp(Exception, 'This user already owns this'):
+            rights_manager.assign_role_for_exploration(
+                self.user_a, self.EXP_ID, self.user_id_a,
+                rights_manager.ROLE_OWNER)
+
+    def test_assign_viewer_role_owner(self):
```

Comment:
Missing prepositions, here and below:

test_assign_viewer_**to**_role_owner
test_assign_voice_artist_**to**_role_owner
test_editor_cannot_be_reassigned_**as**_editor

etc. Please fix, otherwise it's hard to read.

Reply:
done